rs/cli/src/ctx.rs

```diff
@@ -196,6 +197,11 @@ impl DreContext {
         }
     }
 
+    pub fn readonly_ic_admin_for_other_network(&self, network: Network) -> IcAdminWrapper {
+        let ic_admin = self.ic_admin();
+        IcAdminWrapper::new(network, ic_admin.ic_admin_bin_path.clone(), true, Neuron::anonymous_neuron(), false)
```

Comment:
can't we just create new ic-admin from scratch instead of creating this dependency and breaking isolation?